packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -1,86 +1,194 @@
+import { SliceMachineStoreType } from "@src/redux/type";
+import { useSelector } from "react-redux";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import {
+  getApiEndpoint,
+  getChangelog,
+  getRepoName,
+} from "@src/modules/environment";
+import { VIDEO_YOUTUBE_PLAYLIST_LINK } from "@lib/consts";
+import { telemetry } from "@src/apiClient";
+// import { getLibraries } from "@src/modules/slices";
 
 const Navigation: FC = () => {
-  const viewport = useWindowSize();
+  const {
+    changelog,
+    repoName,
+    apiEndpoint,
+    // libraries,
+    // updatesViewed,
+    hasSeenTutorialsToolTip,
+  } = useSelector((store: SliceMachineStoreType) => ({
+    changelog: getChangelog(store),
+    repoName: getRepoName(store),
+    apiEndpoint: getApiEndpoint(store),
+    // libraries: getLibraries(store),
+    // updatesViewed: getUpdatesViewed(store),
+    hasSeenTutorialsToolTip: userHasSeenTutorialsToolTip(store),
+  }));
+
+  const { setUpdatesViewed, setSeenTutorialsToolTip } =
+    useSliceMachineActions();
+
   const isOnline = useNetwork();
 
   const { unSyncedSlices, unSyncedCustomTypes } = useUnSyncChanges();
+  const router = useRouter();
+  const handleNavigation = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) => {
+    const href = event.currentTarget.getAttribute("href");
+    if (href !== null && href) void router.push(href);
+  };
+
+  const latestVersion =
+    changelog.versions.length > 0 ? changelog.versions[0] : null;
+
+  const handleChangeLogNavigationFromUpdateBox = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) => {
+    setUpdatesViewed({
+      latest: latestVersion && latestVersion.versionNumber,
+      latestNonBreaking: changelog.latestNonBreakingVersion,
+    });
+    handleNavigation(event);
+  };
 
   const numberOfChanges = unSyncedSlices.length + unSyncedCustomTypes.length;
   const displayNumberOfChanges = numberOfChanges !== 0 && isOnline;
 
-  return (viewport.width as number) < 640 ? (
-    <Mobile
-      links={getNavigationLinks(displayNumberOfChanges, numberOfChanges)}
-    />
-  ) : (
-    <Desktop
-      links={getNavigationLinks(displayNumberOfChanges, numberOfChanges)}
-    />
+  const currentPath = router.asPath;
+  const repoDomain = new URL(apiEndpoint).hostname.replace(".cdn", "");
+  const repoAddress = apiEndpoint.replace(".cdn.", ".").replace("/api/v2", "");
+  return (
+    <SideNav>
+      <SideNavLogo
+        repoName={repoName}
+        repoDomain={repoDomain}
+        href={repoAddress}
+      />
+
+      <SideNavUnorderedListTop>
+        <SideNavListItem>
+          <SideNavLink
+            title="Page types"
+            href="/"
+            active={
+              currentPath === "/" || currentPath.startsWith("/page-types")
+            }
+            onClick={handleNavigation}
+            Icon={(props) => <CUSTOM_TYPES_CONFIG.page.Icon {...props} />}
```

Comment:
This recreates the Icon component on each render, because it's then used as `<Icon/>`